package.json

```diff
@@ -136,15 +138,15 @@
     "prop-types": "^15.6.0",
     "protomaps-leaflet": "^4.1.1",
     "rc-slider": "^9.7.2",
-    "react": "^16.14.0",
+    "react": "^18.3.1",
     "react-anything-sortable": "^1.5.2",
     "react-color": "^2.19.3",
-    "react-datepicker": "0.53.0",
-    "react-dom": "^16.14.0",
+    "react-datepicker": "7.6.0",
```

Comment:
Can the version be `^7.6.0` or does minor revisions of react-datepicker tend to break things?